nvflare/app_common/aggregators/intime_accumulate_model_aggregator.py

```diff
@@ -232,4 +240,7 @@ def aggregate(self, fl_ctx: FLContext) -> Shareable:
             result_dxo_dict.update({key: aggregated_dxo})
         # return collection of DXOs with aggregation results
         collection_dxo = DXO(data_kind=DataKind.COLLECTION, data=result_dxo_dict)
+        collection_dxo.set_meta_prop(
+            MetaKey.PROCESSED_ALGORITHM, result_dxo_dict.get_meta_prop(MetaKey.PROCESSED_ALGORITHM)
```

Comment:
this result_dxo_dict is a dictionary, so it does not have method `get_meta_prop`

Reply:
Thanks. Turns out this addition was unnecessary. Removed. The unit test is fixed now.